### PR TITLE
Add New Featured Artist: Kola Kid

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎵

### CODE REVIEW

File: news/2020-02-05-new-featured-artist-kola-kid.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/67/previews/1869.mp3" type="audio/mpeg">
+</audio>
+
+----
+
+Like always, these tracks and a bunch more are available right now over at [**Kola Kid**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/67), all pre-timed and just waiting for you to press start. Head on over and get cracking!
+
+There were a lot of correct callouts this time around, especially as the astute among us are starting to notice some *definitely not suspicious* patterns emerging. Big-brained frickin' *radar* technically wins this week's callout by correctly guessing nearly a week before the hint actually was released.
```
Comment (Coppertine) [inline news/2020-02-05-new-featured-artist-kola-kid.md]: So who was this BN? that won?

Comment (Coppertine) [reply news/2020-02-05-new-featured-artist-kola-kid.md]: also, that's why mystery quests should be released at sparse times that make it seem like a usual ranking process

Comment (Ephemeralis) [reply news/2020-02-05-new-featured-artist-kola-kid.md]: radar....

Comment (Coppertine) [reply news/2020-02-05-new-featured-artist-kola-kid.md]:
```suggestion
There were a lot of correct callouts this time around, especially as the astute among us are starting to notice some *definitely not suspicious* patterns emerging. Big-brained frickin' [https://osu.ppy.sh/users/7131099](radar) technically wins this week's callout by correctly guessing nearly a week before the hint actually was released.
```

**Pull request merged**